go/enclave/rpc_server.go

```diff
@@ -289,7 +287,7 @@ func serializeEVMError(err error) ([]byte, error) {
 	}
```

Comment:
why not move line 281 in an else here. It would be much easier to follow the logic

Reply:
Following the https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else logic :)

Reply:
that's why I generally dislike style guides ...
They make the code less expressive because they don't take into account the context.

Which one reads better, if this was to be written down and explained?

Option A
1. I don't know anything about an error, so I create a wrapper 
2. If actually, the error was of that type, forget about it

Option B
1. Check type of the error
2. If it needs wrapping, then wrap

Reply:
I'm just used to avoiding the else when the var is written on both branches. But I get your point it can be less expressive. Changed 👍

Reply:
thanks. this is a problem in go alone. In any other programming language this would not be a debate.

It would be:

```golang

errReturn:= if ok := errors.As(err, &e); ok {		  e	} else {		 evm.SerialisableError{Err: err.Error()}	}
```